kernel/src/mm/ptguards.rs

```diff
@@ -66,10 +80,22 @@ impl PerCPUPageMappingGuard {
         })
     }
 
+    /// Creates a new [`PerCPUPageMappingGuard`] for a 4KB page at the
+    /// specified physical address.
+    ///
+    /// # Arguments
+    ///
+    /// * `paddr` - The physical address for the 4KB page.
+    ///
+    /// # Returns
+    ///
+    /// A `Result` containing the [`PerCPUPageMappingGuard`] if successful,
+    /// or an `SvsmError` if an error occurs.
     pub fn create_4k(paddr: PhysAddr) -> Result<Self, SvsmError> {
         Self::create(paddr, paddr + PAGE_SIZE, 0)
     }
 
+    /// Returns the virtual address associated with the guard.
```

Comment:
This sounds like it's the address of the guard object.

```suggestion
    /// Returns the virtual address for the new mapping.
```